frontend/src/modules/Metadata_Forms/components/MetadataFormFields.js

```diff
@@ -0,0 +1,424 @@
+              onClick={addField}
+              type="button"
+            >
+              Add field
+            </Button>
+          </TableCell>
+        </TableRow>
+        {localFields.length === 0 ? (
+          <TableRow>
+            <TableCell colSpan={6} align="center">
+              No fields found
+            </TableCell>
+          </TableRow>
+        ) : (
+          localFields.map((field, index) => (
+            <TableRow
+              sx={{
+                backgroundColor: field.deleted ? 'whitesmoke' : 'white'
+              }}
+            >
+              <TableCell>
+                <Checkbox
+                  defaultChecked={field.required}
+                  disabled={field.deleted}
+                  onChange={(event) => {
+                    updateField(index, 'required', event.target.value === 'on');
+                  }}
+                />
+              </TableCell>
+              <TableCell>
+                <TextField
+                  disabled={field.deleted}
+                  defaultValue={field.name}
+                  onKeyUp={(event) => {
+                    updateField(index, 'name', event.target.value);
+                  }}
+                  sx={{ width: '100%' }}
+                />
+              </TableCell>
+              <TableCell>
+                <Autocomplete
+                  disablePortal
+                  disabled={field.deleted}
+                  options={fieldTypeOptions.map((option) => option.value)}
+                  defaultValue={field.type}
+                  onKeyUp={(event, value) => {
+                    updateField(
+                      index,
+                      'type',
+                      value || fieldTypeOptions[0].value
+                    );
+                  }}
+                  renderInput={(params) => (
+                    <TextField
+                      sx={{ minWidth: '150px' }}
+                      {...params}
+                      label="Type"
+                      variant="outlined"
+                    />
+                  )}
+                />
+              </TableCell>
+              <TableCell>
+                <TextField
+                  disabled={field.deleted}
+                  defaultValue={field.description}
+                  sx={{ width: '100%' }}
+                  onKeyUp={(event) => {
+                    updateField(index, 'description', event.target.value);
+                  }}
+                />
+              </TableCell>
+              <TableCell>
+                <TextField
+                  defaultValue={field.possibleValues.join(', ')}
+                  disabled={field.deleted}
+                  onKeyUp={(event) => {
+                    updateField(
+                      index,
+                      'possibleValues',
+                      event.target.value.split(',')
+                    );
+                  }}
+                  sx={{ width: '100%' }}
+                />
```

Comment:
I think if the expectation is to specify possibleValues as "value1,value2,...." separated by commas we should explicitly call that out on the UI

Alternatively - what if we used `ChipInput` to enter new possible values such as the way we do with tags currently on Forms 

```
                            <ChipInput
                              fullWidth
                              error={Boolean(touched.tags && errors.tags)}
                              helperText={touched.tags && errors.tags}
                              variant="outlined"
                              label="Tags"
                              placeholder="Hit enter after typing value"
                              onChange={(chip) => {
                                setFieldValue('tags', [...chip]);
                              }}
                            />
```

Also then when displaying the possible values to the user in `DisplayTable` can show the tags a `<Chip>` componenets:

```
              <Box sx={{ mt: 1 }}>
                {tags?.map((tag) => (
                  <Chip
                    sx={{ mr: 0.5, mb: 0.5 }}
                    key={tag}
                    label={tag}
                    variant="outlined"
                  />
                ))}
              </Box>
```

Also should we add some type of data type validation to ensure possible values for String types are strings, and so on for each type (i.e. Int, Bool and Glossary where possibleValues must exist in Glossary to be "valid")

++ I see `glossaryNodeUri` is required for Glossary Term Field Types but we have no way of passing `glossaryNodeUri` from FE on `MetadataFormFields` --> `EditTable`

Reply:
1. Chip input is perfect! Thanks!
2. Glossary Node is coming, I remember about it.

Reply:
[dabdb92](https://github.com/data-dot-all/dataall/pull/1455/commits/dabdb92ac7a254d464945de434af34900e8401d2)

Glossary Nodes [776a855](https://github.com/data-dot-all/dataall/pull/1455/commits/776a8555b92acf34ae42408e2310019fe377a2ed)